AirCasting/Notes/NotesTextView.swift

```diff
@@ -10,3 +10,12 @@ func createNoteTextField(binding: Binding<String>) -> some View {
                maxHeight: 200,
                alignment: .topLeading)
 }
+
+func createEditNoteTextField(binding: Binding<String>) -> some View {
```

Comment:
I'm not a fan of adding this separate function and adding the note flag to the TextView. It would be good to keep the TextView a generic class that can be used for any purpose, not just for notes.